src/sentry/incidents/subscription_processor.py

```diff
@@ -327,6 +329,14 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         """
         This is the core processing method utilized when Query Subscription Consumer fetches updates from kafka
         """
+        if features.has(
+            "organizations:workflow-engine-m3-dual-write", self.subscription.project.organization
+        ):
+            data_packet = DataPacket(
+                query_id=self.subscription.snuba_query.id, packet=subscription_update
+            )
+            process_data_sources([data_packet], query_type=self.subscription.type)
```

Comment:
This would be assigned to a variable (probably detectors).

Reply:
`query_type` here will probably be a not-yet-created enum like `DataSourceType` fyi, I'l try to remember to let you know when it exists